tasks/aiida-daemon.yml

```diff
@@ -28,4 +29,19 @@
     masked: false
     daemon-reload: true
     state: restarted
-  when: not (cloud_platform is defined and cloud_platform == 'docker')
+  when: ansible_service_mgr == "systemd"
+
+- block:
+    - name: Check if the AiiDA daemon is running
```

Comment:
Please add a comment to replace this with `verdi daemon status` after aiida 1.1.0
Added PR to fix this https://github.com/aiidateam/aiida-core/pull/3729

Reply:
Fixed.